docs/ref/models/indexes.txt

```diff
@@ -23,7 +23,7 @@ options`_.
 ``Index`` options
 =================
 
-.. class:: Index(fields=[], name=None)
+.. class:: Index(fields=[], name=None, db_tablespace=None)
```

Comment:
Is this still the correct syntax for the docs now, given `..., *, ...` in the code? Not sure how the Python docs handle this.